aptos-move/framework/tests/move_prover_tests.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use move_deps::move_cli::base::prove::run_move_prover;
+use move_deps::move_prover;
+use std::path::PathBuf;
+use tempfile::tempdir;
+
+pub fn path_in_crate<S>(relative: S) -> PathBuf
+where
+    S: Into<String>,
+{
+    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    path.push(relative.into());
+    path
+}
+
+pub fn run_prover_for_pkg(path_to_pkg: impl Into<String>) {
+    let pkg_path = path_in_crate(path_to_pkg);
+    let config = move_deps::move_package::BuildConfig {
+        test_mode: true,
+        install_dir: Some(tempdir().unwrap().path().to_path_buf()),
+        ..Default::default()
+    };
+    run_move_prover(
+        config,
+        &pkg_path,
+        &None,
+        true,
+        move_prover::cli::Options::default(),
+    )
+    .unwrap();
+}
+
+#[ignore]
+#[test]
```

Comment:
Can't we turn those on or do we need a few more fixes? @davidiw's concern AFAICT where about an user example.

Reply:
I see. We currently have one issue which Meng is going to look at soon: https://github.com/aptos-labs/aptos-core/issues/2581.